src/libcommon/log/sentry/sentryhandler.cpp

```diff
@@ -374,6 +399,19 @@ void SentryHandler::updateEffectiveSentryUser(const SentryUser &user) {
     sentry_set_user(userValue);
 }
 
+void SentryHandler::writeEvent(const std::string &eventStr, bool crash) noexcept {
+    auto eventFilePath =
```

Comment:
Could be put into a function that would be then testable.

Reply:
This function is called only for debugging... (when the concerned environment variables are set)

Reply:
Maybe it's overkill to write unit tests for debugging functions. So it's your call. Ideally, debugging code should work as well as production code (we don't want to debug debugging code when we need it work).

Reply:
Done